index.html

```diff
@@ -8,21 +8,19 @@
     <title>Book List</title>
 </head>
 <body>
-    <h1>Awesome Book List!!!</h1>
+    <h1><strong>All awesome Books</strong>  </h1>
     <div id="bookList"></div>
-
-
+    <span class="spanline1"></span>
+    <h1> <strong>Add a new book</strong>  </h1>
     <form id="form">
       <div>
-        <label for="title">Title:</label>
-        <input type="text" id="title" />
+        <input type="text" id="title" placeholder="Title" />
       </div>
       <div>
-        <label for="author">Author:</label>
-        <input type="text" id="author" />
+        <input type="text" id="author" placeholder="Author"/>
       </div>
       <div>
-        <button type="submit">Add Book</button>
+        <button type="submit" class="addbutton">Add</button>
       </div>
     </form>
     
```

Comment:
- [ ] **README.md** file: I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

- [ ] To make your documentation be more descriptive and professional, I think it would be better with some information such as of how to `clone` your project, `access` the cloned directory, `run/opnen` your app locally, `Run Tests`(if exists),... If you see that some sections of don't have content to be filled, you can remove them. You can refer this **[example](https://gist.github.com/tresorsawasawa/9e65422376a395c17d79bcb10c31077e)** to help you how to fill them.